sirepo/package_data/static/json/srw-schema.json

```diff
@@ -834,11 +838,7 @@
         },
         "multiElectronAnimation": {
             "_super": ["_", "model", "_INTENSITY_REPORT"],
-            "jobRunMode": ["Execution Mode", "JobRunMode", "parallel"],
-            "sbatchCores": ["Cores", "Integer", 128],
-            "sbatchHours": ["Hours", "Float", 0.4],
-            "sbatchQueue": ["Queue", "NERSCQueue", "debug"],
-            "sbatchProject": ["Project", "OptionalString", ""],
+            "_super": ["_", "model", "sbatchFields"],
```

Comment:
I don't think having two _super keys works. At the very least the view is messed up (see screenshot). To replicate yourself start a multiElectronAnimation -> pencil icon -> view tab. There may be other problems.

![Screen Shot 2022-08-16 at 2 03 22 PM](https://user-images.githubusercontent.com/10264515/184974317-450f5a49-9a83-427b-b4c0-2f37318d0122.png)

Reply:
got rid of the two supers